dts/bindings/rng/virtio,device4.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2025 TOKITA Hiroshi
+# SPDX-License-Identifier: Apache-2.0
+
+description: VirtIO Entropy device (ID:4)
+
+compatible: "virtio,device4"
```

Comment:
Just wondering if this is the optimal compatible string. Is this already a conventional compatible string that is used elsewhere? Is there a more descriptive one?

Reply:
yep, that is a good point. The name looks very unconventional  but I don't have a better option :)